### PR TITLE
Fix incorrect resolution string in config name

### PR DESCRIPTION
**Pull request by SBCV**
Rename 

`configs/segmenter/segmenter_vit-l_mask_8x1_512x512_160k_ade20k.py`

to

`configs/segmenter/segmenter_vit-l_mask_8x1_640x640_160k_ade20k.py`,

since it expects a resolution of 640x640.

### CODE REVIEW

**Comment by linfangjian01** [comment]
Thank you for your contribution，please fix lint.

**Comment by SBCV** [comment]
How? All I did was renaming the file. This looks to me like a bug in the linter, no?

**Comment by SBCV** [comment]
Hey, sorry for the late reply - was on vacation and had no access to my desktop pc. 
Thank you for pointing out the tutorial and adapting the changes for the linter.

File: configs/segmenter/segmenter.yml [hunk 1]
```diff
@@ -121,5 +121,5 @@ Models:
     Metrics:
       mIoU: 52.16
       mIoU(ms+flip): 53.65
-  Config: ''
-  Weights: ''
+  Config: configs/segmenter/segmenter_vit-l_mask_8x1_640x640_160k_ade20k.py
+  Weights: https://download.openmmlab.com/mmsegmentation/v0.5/segmenter/segmenter_vit-l_mask_8x1_512x512_160k_ade20k/segmenter_vit-l_mask_8x1_512x512_160k_ade20k_20220105_162750-7ef345be.pth
```
Comment (SBCV) [inline configs/segmenter/segmenter.yml:125]: Shouldn't this URL also be changed? To reflect the resolution of 640x640, it should be something like this:

https://download.openmmlab.com/mmsegmentation/v0.5/segmenter/segmenter_vit-l_mask_8x1_640x640_160k_ade20k/segmenter_vit-l_mask_8x1_640x640_160k_ade20k_20220105_162750-7ef345be.pth

Comment (MeowZheng) [reply configs/segmenter/segmenter.yml:125]: Thanks for this kind reminder, we will rename the URL later, and create another pr to fix it.

**Review by MeowZheng — COMMENTED** [review]
Please resolve conflicts.

**Comment by MeowZheng** [comment]
We fixed the problem about `segmenter` in #1695, and close this pr.

**Pull request closed without merging**